pkg/trace/otel.go

```diff
@@ -199,3 +200,11 @@ func (t *otelTracer) spanID(ctx context.Context) string {
 func (t *otelTracer) parentID(ctx context.Context) string {
 	return ""
 }
+
+func (t *otelTracer) setForce(force bool) {
+	t.force = force
+}
```

Comment:
In at least *some* of the tracing implementations, I tried to make "force-trace" a string rather than a boolean, then include that string value as a trace tag.

That way, I can do `curl -H 'X-Force-Trace: rltesthackstuff' ...` then go look for traces in the Honeycomb UI that have "force_trace == rltesthackstuff".  

(Admittedly, I don't think I've been consistent about this, and some services do still flatten this to a bool.  I haven't gotten around to fixing it all yet.  Still, when it works, it's pretty great.)

Reply:
Oh I like that idea! I'll update things to support this.

Reply:
Stored as a string and forces on any non empty string.